src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -2856,26 +2863,25 @@ void CheckPlantOnAbort(EnergyPlusData &state)
                     for (CompNum = 1; CompNum <= state.dataPlnt->PlantLoop(LoopNum).LoopSide(SideNum).Branch(BranchNum).TotalComponents; ++CompNum) {
                         ShouldBeACTIVE = false;
```

Comment:
Yikes!!

Reply:
I'm not sure what I was thinking here exactly. I think I was referring to the long array lookup chain in the loop test. Just ignore this for now.